application/src/main/java/org/thingsboard/server/service/entitiy/DefaultTbNotificationEntityService.java

```diff
@@ -68,12 +67,14 @@ public <E extends HasName, I extends EntityId> void notifyEntity(TenantId tenant
     }
 
     @Override
-    public <E extends HasName, I extends EntityId> void notifyDeleteEntity(TenantId tenantId, I entityId, E entity,
+    public <E extends HasName, I extends EntityId> void notifyDeleteEntity(TenantId tenantId, I entityId, E entity, EntityId originatorId,
```

Comment:
Create own method for alarm (notifyDeleteAlarm)

Reply:
ok